CONTRIBUTING.md

```diff
@@ -227,6 +228,10 @@ Finally, add all other arguments to the config file. There are a few exceptions:
 
 * Arguments related to printing the information such as printing the version (`-v`, `--version`) or printing the help (`-h`, `--help`) should not be added to the config file.
 
+Note:
+
+* If the documentation mentions a default value for some of the arguments, please don't add them as default values in the Viash config, but instead as an example. This is because the documentation of the tool might be incorrect or outdated, and by adding a default value to the Viash config we might be changing the default behaviour of the tool. For example, do not write `name: --my_arg, description: "<description>", default: 10`, but instead `name: --my_arg, description: "<description>. Default: 10.", example: 10`.
```

Comment:
This will be problematic when automatically generating documentation for components, right? Because the default will be unstructured? And there might be cases where you might want to add extra or more specific examples?

Reply:
A partial solution could be to pass multiple examples as a list?

Reply:
To state the obvious: We can not turn something unpredictable (a changing default value in a tool we wrap) into something predictable with Viash. So there's a choice to be made:

1. do we _always_ fall back to the underlying tool's default, or
2. do we specify our own default value which makes sense

I'm torn between these two myself.

One argument against the situation currently described in the guidelines is that when using a biobase component/module in a pipeline it does not suffice to look at the Viash config for the component. Rather, one should look at the documentation of the underlying tool. Unless, that is, the default value is documented in the description of the argument. But then that has to be kept up-to-date as well!